test/libyul/yulOptimizerTests/fullSuite/clear_after_if_continue.yul

```diff
@@ -0,0 +1,22 @@
+{
+    let y := mload(0x20)
+    for {} and(y, 8) { if y { revert(0, 0) } } {
+        if y { continue }
+        sstore(1, y)
+    }
+    if y { revert(0, 0) }
+}
+// ====
+// step: fullSuite
+// ----
+// {
+//     {
+//         let y := mload(0x20)
+//         for { } and(y, 8) { if y { revert(0, 0) } }
```

Comment:
Why doesn't it see that `y = 0` here?

Reply:
It's the full suite, the `y=0` is removed again.

Reply:
Yes, but what I meant is that why doesn't it (some optimizer) see that `y = 0` at post and removes the `if` entirely?

Reply:
I still do not fully understand. There is `if y {continue}`, so the post-block is executed in both situations `y=0` and `y!=0`.